doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -299,7 +299,7 @@ Fields used as identifiers in the NGSIv2 API follow special rules regarding allo
 
 The rules are:
 
-* Allowed characters are the ones in the plain ASCII set except the following ones: whitespace, `&`, ?`, `/` and `#`.
+* Allowed characters are the ones in the plain ASCII set except the following ones: whitespace, `&`, `?`, `/` and `#`.
```

Comment:
Not actually related with this PR, but a small typo known time ago now being fixed. NTC